packages/entities/entities-upstreams-targets/src/components/UpstreamsForm.vue

```diff
@@ -235,144 +244,55 @@ const fetchErrorHandler = (err: AxiosError): void => {
   emit('error', err)
 }
 
-const getPayload = computed((): UpstreamFormPayload => {
-  const result: UpstreamFormPayload = {
-    name: state.fields.name,
-    slots: Number(state.fields.slots),
-    algorithm: state.fields.algorithm,
-    hash_on: state.fields.hashOn,
-    hash_fallback: state.fields.hashFallback,
-    healthchecks: {},
-  }
-
-  if (state.fields.hostHeader) {
-    result.host_header = state.fields.hostHeader
-  }
-
-  if (state.fields.clientCertificate) {
-    result.client_certificate = { id: state.fields.clientCertificate }
-  }
-
-  if (state.fields.healthchecksThreshold) {
-    result.healthchecks.threshold = Number(state.fields.healthchecksThreshold)
-  }
-
-  if (state.fields.tags) {
-    result.tags = state.fields.tags.split(',')?.map((tag: string) => String(tag || '')
-      .trim())?.filter((tag: string) => tag !== '')
-  }
+const getBasePayload = (): UpstreamFormPayload => ( {
+  name: state.fields.name,
+  slots: Number(state.fields.slots),
+  algorithm: state.fields.algorithm,
+  hash_on: state.fields.hashOn,
+  hash_fallback: state.fields.hashFallback,
+  healthchecks: {
+    threshold : Number(state.fields.healthchecksThreshold || '0'),
+  },
+  host_header : state.fields.hostHeader || null,
+  client_certificate : state.fields.clientCertificate ? { id: state.fields.clientCertificate } : null,
+})
 
+const getHashingConfig = (payload: UpstreamFormPayload): void => {
```

Comment:
This is a function with side effects (e.g., updating fields in the `payload` from the parameter) but named `getHashingConfig`. Could we consider either:

* Make this function return a value like `Partial<UpstreamFormPayload>` (i.e., a Record contains `hash_on_header`, `hash_on_cookie`, etc.), and update the `payload` in the `getPayload` function below
* Rename this function to something like `updateHashConfig` to reflect its possible side effects

Reply:
Done. Please review it again.